src/app/content/highlights/components/EditCard.tsx

```diff
@@ -69,6 +71,16 @@ const EditCard = React.forwardRef<HTMLElement, Props>((
     }
   };
 
+  React.useEffect(() => {
+    if (!isFocused) { return; }
```

Comment:
this will still log pass when it shouldn't once you do the thing about showing all cards. i think the actual check you want is to see if `data` is defined